Assignment3.java

```diff
@@ -3,13 +3,24 @@
 public class Assignment3 {
```

Comment:
maybe I'm not right? but isn't it better to name your class smth like `WordsGridFinder` ? To make it more understandable .